package.json

```diff
@@ -28,27 +27,21 @@
   "devDependencies": {
     "bower": "^1.8.2",
     "broccoli-asset-rev": "^2.4.5",
-    "ember-ajax": "^2.4.1",
+    "ember-ajax": "^2.5.6",
     "ember-cli": "2.12.3",
-    "ember-cli-chai": "^0.4.3",
     "ember-cli-dependency-checker": "^1.3.0",
     "ember-cli-frost-blueprints": "^4.0.0",
```

Comment:
Do we have any content in this repo that we would have created using the blueprints provided by this, or anticipate using such content? Do we want to keep it for consistency or remove it?  Think we might have had this conversation already...

Reply:
I wanted to keep it in case we need to add some tests at some point